presto-main/src/test/java/io/prestosql/sql/planner/optimizations/TestAddExchangesPlans.java

```diff
@@ -428,6 +428,17 @@ public void testExchangesAroundTrivialProjection()
                 anyTree(
                         project(
                                 values("a"))));
+
+        assertPlan(
+                "SELECT 1 UNION ALL SELECT 1",
+                anyTree(
+                        exchange(
+                                LOCAL,
+                                REPARTITION,
```

Comment:
What we repartition on? is it round robin in disguise?

Reply:
Yes :(

Reply:
uh. maybe worth adding a comment for people coming here for the first (second, third, ...) time, or somehow reflect that in `exchange` matcher instantiation